lib/generate_localized.dart

```diff
@@ -139,7 +139,7 @@ class MessageGeneration {
       // 24356
       """
   final messages = _notInlinedMessages(_notInlinedMessages);
-  static _notInlinedMessages(_) => <String, Function> {
+  static _notInlinedMessages(_) => <String, dynamic> {
```

Comment:
Thanks. A couple of questions/comments. 
  - Consider using --json for the message generation. We've switched to that as the default internally because it's not nearly as hard on the compiler.
  - Where is this lint coming from? There seems to be an analyzer warning of that form, but I don't see it I just generate the code. And I don't understand why it would be wrong. This looks like a Map<String, Function>, and that's what the keys and values are?

Reply:
Thanks for the additional details.

It looks like I already have a better fix queued up, which is giving a better return type to simpleMessage  ( String Function() ) . I think that should also address this lint, and I should publish a newer version of Intl.

More generally, consider excluding generated code from the lint rules. It's not helpful for you if you don't control the code, and we can't satisfy all possible lint rules that users have enabled. Some of them actually contradict each other.

Reply:
As far as I am aware, there is no // ignore for strong mode settings, this is only for linters.
This error is caused by implicit-dynamic: false, which is no lint, but a setting of the analyzer, that actually changes how the type system is checked.

I agree that generated files should be exempt from such settings.
But there is this bug of the dartanalyzer:
dart-lang/sdk#25551

Which causes this to still show up as errors in IDEs (if you happen open the .g.dart file). Android Studio also gives you warnings on Hot Reloading if this happens, and the only way to fix this is to close the .g.dart file, close android studio and open it again.

Now, obviously this should be fixed in the dartanalyzer. However, this bug has been open for almost four years and is unlikely to be fixed soon.

Reply:
Correction, there is a setting for ignoring these errors: 
// ignore_for_file: implicit_dynamic_parameter
should fix it